<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AmendOffenseDialog</class>
 <widget class="QDialog" name="AmendOffenseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>376</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>100</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Palatino Linotype</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Amendment</string>
  </property>
  <widget class="QFrame" name="case_information_frame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>981</width>
     <height>131</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>800</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QLabel" name="defendant_name_label">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>53</y>
      <width>831</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="case_number_label">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>13</y>
      <width>831</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="defendant_attorney_name_label">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>93</y>
      <width>821</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>981</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <widget class="QPushButton" name="continueButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>130</y>
      <width>161</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>801</x>
      <y>30</y>
      <width>161</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>614</width>
      <height>152</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Prosecutor motion to amend is </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="offense_choice_box">
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Granted</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Denied</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>The charge of </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>is amended to</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>pursuant to</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBox_3">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>801</x>
      <y>80</y>
      <width>161</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Fields</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>offense_choice_box</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>comboBox_3</tabstop>
  <tabstop>continueButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>continueButton</sender>
   <signal>clicked()</signal>
   <receiver>AmendOffenseDialog</receiver>
   <slot>proceed_to_ability_to_pay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>810</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>continueButton</sender>
   <signal>released()</signal>
   <receiver>AmendOffenseDialog</receiver>
   <slot>close_window()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>810</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>AmendOffenseDialog</receiver>
   <slot>close_window()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>830</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>853</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>comboBox</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>844</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>comboBox_2</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>823</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>606</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>comboBox_3</receiver>
   <slot>clearEditText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>812</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>612</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>proceed_to_ability_to_pay()</slot>
  <slot>close_window()</slot>
  <slot>add_offense()</slot>
 </slots>
</ui>
